data/strings/strings.txt

```diff
@@ -22562,7 +22565,7 @@
 
   [download_map_carplay]
     tags = ios
-    en = Please download maps in Organic Maps app on your device
+    en = Please download maps in the Organic Maps app on your mobile device
```

Comment:
A shorter version on smaller screens is preferrable.

Reply:
Yes, shorter is better.  The point is "Maps can only be downloaded on the Android device." but I don't know the carplay term for that.

Reply:
fixed by changing to `Please download maps in the app on your mobile device`